data/Vaaran Bhai Gurdas Ji/41.json

```diff
@@ -125,7 +125,7 @@
         "id": "MGZC",
         "source_page": 41,
         "source_line": 1,
-        "gurmukhi": "guru ismr mnweI kwlkw; KMfy kI vylw [",
+        "gurmukhi": "guru dws mnweI kwlkw; KMfy kI vylw [",
```

Comment:
Should be `ismir`

Reply:
this was a mistake, i forgot to discard changes before committing the next line, will do properly now

Reply:
https://github.com/ShabadOS/database/pull/1009/commits/29f0ffef498ed65b7c81fc3c3e378c057bf8c94c & https://github.com/ShabadOS/database/pull/1009/commits/2b7d3811543ce29ed1601dc3b46affcb6b889b61